### PR TITLE
Update running-on-kubernetes-standalone.asciidoc

### PR DESCRIPTION
**Pull request by leahleahy**

### CODE REVIEW

File: docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc [hunk 1]
```diff
@@ -64,10 +62,10 @@ User need to set Elasticsearch settings before deploying the manifest:
 [float]
 ===== Running {agent} on master nodes
 
-Kubernetes master nodes can use https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/[taints]
+Kubernetes master nodes use https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/[taints]
 to limit the workloads that can run on them. The manifest for standalone {agent} defines
-tolerations so as to run on master nodes too and being able to collect metrics from the control plane
-components of Kuberentes (scheduler, controller manager)
+tolerations to run on master nodes, which collects metrics from the control plane
+components of Kuberentes (scheduler, controller manager).
```
Comment (leahleahy) [inline docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc:68]: I want to make sure this is rephrased correctly!

File: docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc [hunk 2]
```diff
@@ -103,9 +101,9 @@ elastic-agent-hs4pb             1/1     Running   0          81m
 [float]
 ==== Autodiscover targeted Pods
```
Comment (leahleahy) [inline docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc:102]: Same question as the previous article, but should autodiscover be hyphenated?

File: docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc [hunk 3]
```diff
@@ -2,26 +2,25 @@
...
 
-You deploy {agent} in two different ways at the same time:
+Deploy {agent} in two different ways at the same time:
 
 * As a https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/[DaemonSet]
 to ensure that there's a running instance on each node of the cluster. These
-instances are used to retrieve most metrics from the host, such as system
-metrics, Docker stats, and metrics from all the services running on top of
+instances are used to retrieve metrics from the host, such as system
+metrics, Docker stats, and metrics from all the services running on
```
Comment (bmorelli25) [inline docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc:23]: This changes the meaning too much. Not all metrics are collected from the hosts, and it is appropriate to say running on top of Kubernetes vs running on Kubernetes

```suggestion
instances are used to retrieve most metrics from the host, such as system
metrics, Docker stats, and metrics from all the services running on top of
```

File: docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc (hunk 1 above)
Comment (bmorelli25) [reply docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc:68]: Looks good

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

File: docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc (hunk 2 above)
Comment (EamonnTP) [reply docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc:102]: No need to hyphenate.

File: docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc [hunk 4]
```diff
@@ -103,9 +101,9 @@ elastic-agent-hs4pb             1/1     Running   0          81m
 [float]
 ==== Autodiscover targeted Pods
 
-It is possible to define autodiscover conditions so as to allow {agent} to automatically
-identify Pods and start collecting from them using predefined integrations. For example if a
-user want to automatically identify a Redis Pod and start monitoring it using the Redis integration
+Ddefine the autodiscover conditions to allow {agent} to automatically
```
Comment (bmorelli25) [inline docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc:104]:
```suggestion
You can define autodiscover conditions to allow {agent} to automatically
```

**Review by bmorelli25 — APPROVED** [review]

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Pull request merged**